src/Kunstmaan/MediaBundle/Helper/File/FileHandler.php

```diff
@@ -180,12 +184,12 @@ public function prepareMedia(Media $media)
             $media->setContent($file);
         }
 
-        $contentType = $this->mimeTypeGuesser->guess($content->getPathname());
+        $contentType = $this->mimeTypeGuesser->guessMimeType($content->getPathname());
```

Comment:
I think we should move this mimeTypeGuesser call to a private function, where we can check if it is the old or new class and then call `guess` of `guessMimeType`. Same for the other calls bellow